synapse/app/synctl.py

```diff
@@ -23,21 +23,41 @@
 import subprocess
 import sys
 import yaml
+import errno
+import time
 
 SYNAPSE = [sys.executable, "-B", "-m", "synapse.app.homeserver"]
 
 GREEN = "\x1b[1;32m"
+YELLOW = "\x1b[1;33m"
 RED = "\x1b[1;31m"
 NORMAL = "\x1b[m"
 
 
+def pid_running(pid):
+    try:
+        os.kill(pid, 0)
+    except OSError, err:
+        if err.errno != errno.EPERM:
+            return False
+        else:
+            return True
+    else:
+        return True
```

Comment:
Apparently, as I read, it is considered to be good programming habits for big code bases.
But I understand , It's unnecessary here.